sidebars.js

```diff
@@ -16,6 +16,7 @@ module.exports = {
         "label": "Getting Started",
         "items": [
           "getting-started",
+          "okteto-license",
```

Comment:
The current getting started is still oriented to Cloud and the Okteto CLI, would it be better to add this doc as part of the `SaaS vs. Self-Hosted` section?

Reply:
I'm open to that, for sure.

I wasn't sure where it could live but landed here so I could open the PR and start a discussion.

I think living within SaaS vs. Self-Hosted could make sense. 

@rberrelleza any opinion on this?

Reply:
I agree it doesn't fit in the getting started section, since that is focused at the developer.   I think it fits in the same section than SaaS vs Self-Hosted. But I like what gitlab does for the 'subscribe' section. Maybe we can quickly do the same? (even if now we only have these two)  -> https://docs.gitlab.com/ee/subscriptions/

Another option is to put it inside the 'Admin' section, since only admins care about the license.

Reply:
My vote would be add it to the `Admin` section for now. It's a simpler/faster change (I think) and gets this PR merged sooner (assuming I don't lose track of it again 🤦‍♂️)